packages/atlas/src/components/_overlays/SendTransferDialogs/WithdrawDialog.tsx

```diff
@@ -0,0 +1,119 @@
+import { FC, useState } from 'react'
+
+import { FullMembershipFieldsFragment } from '@/api/queries'
+import { Avatar } from '@/components/Avatar'
+import { NumberFormat } from '@/components/NumberFormat'
+import { Text } from '@/components/Text'
+import { JoyTokenIcon } from '@/components/_icons/JoyTokenIcon'
+import { FormField } from '@/components/_inputs/FormField'
+import { Input } from '@/components/_inputs/Input'
+import { DialogModal } from '@/components/_overlays/DialogModal'
+import { JOY_CURRENCY_TICKER } from '@/config/token'
+import { useTokenPrice } from '@/providers/joystream'
+
+import { Fee } from './Fee'
+import { BalanceWrapper, DestinationAccountWrapper, Summary, SummaryRow } from './SendTransferDialogs.styles'
+
+type WithdrawDialogProps = {
+  onExitClick: () => void
+  accountBalance?: number
+  activeMembership?: FullMembershipFieldsFragment | null
+  show: boolean
+  avatarUrl?: string | null
+  channelBalance?: number
+}
+
+const ADDRESS_CHARACTERS_LIMIT = 4
+
+export const WithdrawDialog: FC<WithdrawDialogProps> = ({
+  onExitClick,
+  accountBalance = 0,
+  activeMembership,
+  show,
+  avatarUrl,
+  channelBalance = 0,
+}) => {
+  const [amount, setAmount] = useState<number | null>(null)
+  const { convertToUSD } = useTokenPrice()
+  const convertedAmount = convertToUSD(amount || 0)
+  const [error, setError] = useState<string>()
+
+  const handleWithdraw = () => {
```

Comment:
Let's use `useForm` here. Validation should work on submit and after you submit with wrong values, the mode of validation should change to `onChange`(I think this is the default behavior of react-hook-form).
Right now, after you submit and you focus on the input, the error stays until you click submit again, which is not intuitive in my opinion.